stress/src/throughput.rs

```diff
@@ -69,8 +69,8 @@ where
     let worker_stats_shared_monitor = Arc::clone(&worker_stats_shared);
 
     let handle_main_thread = thread::spawn(move || {
-        let mut start_time = Instant::now();
-        let mut end_time = start_time;
+        let mut last_collect_time = Instant::now();
+        let mut current_time: Instant;
```

Comment:
nit - do we need to declare it here, or can be done inside loop?

Reply:
I have moved the `current_time` variable inside the loop.

> I believe cost of creating is negligible if we are trying to avoid that.

Yeah, since we would always initialize it to `Instant::now()`, most likely the compiler should optimize it.